### DISCUSSION TITLE
Is Asking for Private Key Password

### DISCUSSION THREAD

**Discussion by greatvc** [pivpn/pivpn#1871] [Support] [answered]
Hi, i haven't connected via ios or android in a long time so i assume it is an update that happened to OPENVPN client.

I have a rasberry pi with pivpn, i use always the same profile .ovpn
This is still working on my Windows OpenVPN client assume that because it has never been updated (3.4.4) so there i can connect fin to the pivpn server.

On the androif and ios it asks for a private key password. I do not have that. What can i do to fix it easily, change something in a conf to accept legacy connections or something like that ?

Thanks
Vassilis

**Comment by orazioedoardo** [comment]
Try to recreate a client with the nopass option.

**Reply by greatvc** [reply]
thanks for the reply!

the nopass refers to the private key password or to the VPN password, cause i like to have a vpn password for connection.

Also if i create a new one the old will be lost, i'm remote and i was wondering if i can do that from the VPN cpnnection and basically create a second package of .ovpn

**Reply by orazioedoardo** [reply]
I'm assuming the password you are being asked for is the .ovpn encryption password, there is no other password. 

The nopass option doesn't encrypt the file. You can create a second .ovpn in PiVPN with a different name.

**Comment by greatvc** [comment, marked as answer]
Thanks, i created a new completely profile and it worked. The option with nopass is without a VPN password. Here i'm talking aobut the "Private Key Password" which is something that the new openvpn client was asking in ios and android.

I created two new profiles from pivpn one with nopass and other normal. Both are worked. Don't know why.
